packages/ui/src/council/modals/RecoverVoteStake/RecoverVoteStakeSignModal.tsx

```diff
@@ -0,0 +1,87 @@
+import { SubmittableExtrinsic } from '@polkadot/api/types'
+import { ISubmittableResult } from '@polkadot/types/types'
+import React from 'react'
+import { ActorRef, State } from 'xstate'
+
+import { AccountLockInfo, lockInfoLayout } from '@/accounts/components/AccountLockInfo'
+import { useMyAccounts } from '@/accounts/hooks/useMyAccounts'
+import { accountOrNamed } from '@/accounts/model/accountOrNamed'
+import { ButtonPrimary } from '@/common/components/buttons'
+import { InputComponent } from '@/common/components/forms'
+import { Arrow } from '@/common/components/icons'
+import { ListHeader, ListHeaders } from '@/common/components/List/ListHeader'
+import { ModalBody, ModalFooter, TransactionInfoContainer } from '@/common/components/Modal'
+import { RowGapBlock } from '@/common/components/page/PageContent'
+import { TransactionInfo } from '@/common/components/TransactionInfo'
+import { TextMedium, TokenValue } from '@/common/components/typography'
+import { useModal } from '@/common/hooks/useModal'
+import { useSignAndSendTransaction } from '@/common/hooks/useSignAndSendTransaction'
+import { TransactionModal } from '@/common/modals/TransactionModal'
+import { TransactionEvent } from '@/common/model/machines'
+import { TransactionContext } from '@/proposals/modals/AddNewProposal/machine'
+
+import { RecoverVoteStakeModalCall } from '.'
+
+interface Props {
+  service: ActorRef<TransactionEvent, State<TransactionContext>>
+  transaction: SubmittableExtrinsic<'rxjs', ISubmittableResult>
+}
+
+export const RecoverVoteStakeSignModal = ({ service, transaction }: Props) => {
+  const {
+    hideModal,
+    modalData: { address, stake },
+  } = useModal<RecoverVoteStakeModalCall>()
+  const { allAccounts } = useMyAccounts()
+
+  const { sign, isReady, paymentInfo } = useSignAndSendTransaction({
+    service,
+    transaction,
+    signer: address,
+  })
+
+  return (
+    <TransactionModal onClose={hideModal} service={service}>
+      <ModalBody>
+        <TextMedium light>
+          You intend to recover <TokenValue value={stake} /> stake locks from account.
+        </TextMedium>
+
+        <RowGapBlock gap={8}>
+          <ListHeaders $colLayout={lockInfoLayout}>
+            <div />
```

Comment:
The `<div />` is there to fill the first column, as only the 2nd and the 3rd have headers in the design. Not sure how to do it better and still use the same `$colLayout` between headers and row.

Reply:
OK, that's fine. Maybe create a `<EmptyHeader />` component for this (in common, where other list components are present) so we would have a clear understanding of what it is. Even though, it will be an empty `<div />` it will be fluent enough.